kura/examples/org.eclipse.kura.example.ble.tisensortag.dbus/OSGI-INF/metatype/org.eclipse.kura.example.ble.tisensortag.dbus.BluetoothLe.xml

```diff
@@ -49,8 +49,8 @@
             default="120"
             description="Period in seconds between 2 publishes. It must be greater than scan_time."/>
             
-        <AD id="enableTermometer"
-            name="enableTermometer"
+        <AD id="enableTemperature"
+            name="enableTemperature"
             type="Boolean"
```

Comment:
The `termometer` is for sure a typo... but in all others properties we use the name of the tool/sensor used to read that physical quantity. So, we can use `thermometer` or we can decide to replace all properties name with the name of the quantity.